src/labels/CustomLabels.labels

```diff
@@ -1087,6 +1087,30 @@
         <shortDescription>Field label for Accounts and Contacts Settings</shortDescription>
         <value>Disable Preferred Phone Enforcement</value>
     </labels>
+    <labels>
+        <fullName>stgHCPicklistValueInactive</fullName>
```

Comment:
Do we need to include translation files for these labels?

Reply:
Good point. We really do.